File: benchmark/corpus/03_div_zero/a_share_per_person.js
// Splits a bill by a head count read from input; the count can be 0.
function share(total, count) {
  return Math.floor(total / count); // DANGER
}

const line = (await new Response(Deno.stdin.readable).text()).trim();
const count = Number(line);
if (Number.isNaN(count)) console.log("not a number: " + line);
else console.log("each pays " + share(59500, count));

File: benchmark/corpus/03_div_zero/a_share_per_person.py
# Splits a bill by a head count read from input; the count can be 0.
import sys


def share(total, count):
    return total // count  # DANGER


line = sys.stdin.readline().strip()
try:
    count = int(line)
except ValueError:
    print("not a number: " + line)
else:
    print("each pays %d" % share(59500, count))
